tests/e2e/setup.go

```diff
@@ -27,28 +27,38 @@ import (
 type CCVTestSuite struct {
 	suite.Suite
 	coordinator   *ibctesting.Coordinator
+	setupCallback SetupCallback
+
 	providerChain *ibctesting.TestChain
-	consumerChain *ibctesting.TestChain
 	providerApp   e2eutil.ProviderApp
-	consumerApp   e2eutil.ConsumerApp
-	path          *ibctesting.Path
-	transferPath  *ibctesting.Path
-	setupCallback SetupCallback
-	skippedTests  map[string]bool
+
+	// The first consumer chain among multiple.
+	consumerChain *ibctesting.TestChain
```

Comment:
We can eventually remove these fields in favor of accessing all this state via `consumerBundles` below. However, this will require large diffs, and is not feasible to do in a single PR